### PR TITLE
Update Beta branch with new branding.

### PR DESCRIPTION
**Pull request by kellyjandrews**
## Description
This pull requests updates the beta branch to match the current changes in master. 

## Motivation and Context
Resolves the beta branch for the brand changes.

## Testing Details
All tests pass, and have been updated to reflect new branding.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -40,7 +40,11 @@ This Code of Conduct applies within all community spaces, and also applies when
 
 ## Enforcement
 
+<<<<<<< HEAD
```
Comment (dragonmantank) [inline CODE_OF_CONDUCT.md]: Conflict markers got merged

File: src/Dispatch.js [hunk 2]
```diff
@@ -14,9 +12,9 @@ class Dispatch {
     this.options = options;
 
     // Used to facilitate testing of the call to the underlying object
-    this._nexmo = this.options.nexmoOverride || nexmo;
+    this._vonage = this.options.vonageOverride || vonage;
```
Comment (dragonmantank) [inline src/Dispatch.js:15]: Does this need removed from here like with the other modules?

This is a rebranding branch, but might be good to flag as an additional change if needed.

File: test/HttpClient-test.js [hunk 3]
```diff
@@ -414,7 +414,7 @@ describe("HttpClient Object", function() {
   });
 
   it("should allow User-Agent header to be set via options", function() {
-    var expectedUserAgent = "nexmo-node/1.0.0/v4.4.7";
+    var expectedUserAgent = "vonage-node/1.0.0/v4.4.7";
```
Comment (dragonmantank) [inline test/HttpClient-test.js]: Should have this match the actual user agent string to be safe

**Review by dragonmantank — CHANGES_REQUESTED** [review]
Couple of minor things pop out just giving this a once over. Mostly everything looks fine since it's all name swapping, but didn't run any of the code. Was just looking for obvious things.

File: src/Dispatch.js (hunk 2 above)
Comment (kellyjandrews) [reply src/Dispatch.js:15]: So this is something new in the beta branch that I've not seen elsewhere, but it's baked into tests.  I don't think it was removed from the other modules, but actually added to these.

**Review by dragonmantank — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**